docs/src/pages/index.js

```diff
@@ -116,7 +116,15 @@ function SnippetGenerator({ template }) {
         <h2 className={styles.cardHeading} id={template.name}>{template.name}</h2>
       </div>
       {template.beforeDescriptionNote && <p className={styles.cardDescription} dangerouslySetInnerHTML={{__html: template.beforeDescriptionNote}} />}
-      <p className={styles.cardDescription}>{template.description}</p>
+      <p className={styles.cardDescription} dangerouslySetInnerHTML={{ __html: template.description }}/>
+      <div className={styles.cardSpoilerContainer}>
+      {template.spoiler && (
+          <details className={styles.cardSpoilerDetails}>
+            <summary className={styles.cardSpoilerSummary}>Spoiler</summary>
```

Comment:
Was that `Spoiler` consulted with copy team? Sounds weird, I would expect `Preview` or something

Reply:
Agreed. To be changed.